src/iuse.cpp

```diff
@@ -4620,4438 +4623,4341 @@ std::optional<int> iuse::blood_draw( Character *p, item *it, bool, const tripoin
         blood_temp = units::from_celsius( 37 );
         if( p->has_trait( trait_ACIDBLOOD ) ) {
             acid_blood = true;
-        }
-        // From wikipedia,
-        // "To compare, this (volume of blood loss that causes death) is five to eight times
-        // as much blood as people usually give in a blood donation.[2]"
-        // This is half a TU, hence I'm setting it to 1/10th of a lethal exsanguination.
-        p->vitamin_mod( vitamin_redcells, vitamin_redcells->min() / 10 );
-        p->vitamin_mod( vitamin_blood, vitamin_blood->min() / 10 );
-        p->mod_pain( 3 );
-    }
-
-    if( acid_blood ) {
-        item acid( "blood_acid", calendar::turn );
-        acid.set_item_temperature( blood_temp );
-        it->put_in( acid, item_pocket::pocket_type::CONTAINER );
-        if( one_in( 3 ) ) {
-            if( it->inc_damage() ) {
-                p->add_msg_if_player( m_info, _( "…but acidic blood melts the %s, destroying it!" ),
-                                      it->tname() );
-                p->i_rem( it );
-                return 0;
+            if( p->has_trait( trait_VAMPIRE ) ) {
```

Comment:
You have a missing `}` before this, that's why the entire rest of the file changed.